src/lib/UniversalDiscovery/Event/Subscriber/RichTextEmbedAllowedContentTypes.php

```diff
@@ -61,7 +61,12 @@ private function getAllowedContentTypesIdentifiers(): array
             $allowedContentTypesIdentifiers[] = $contentType->identifier;
         }
 
-        return $allowedContentTypesIdentifiers;
+        $allowedContentTypesIdentifiers = count($contentTypesAllowedViaConfig)
+            ? array_intersect($contentTypesAllowedViaConfig, $allowedContentTypesIdentifiers)
+            : $allowedContentTypesIdentifiers;
+
+        //hacky, but as of now null or empty array means UDW allows all Content Types, which shouldn't be the case
+        return empty($allowedContentTypesIdentifiers) ? [microtime()] : array_values($allowedContentTypesIdentifiers);
```

Comment:
This is not acceptable as long-term solution. @dew326 is it possible to handle e.g. `false` value on UDW side as indicator of "none of content types is allowed" ?

Reply:
List of allowed Content Types:
null – all Content Types are allowed,
[] – empty table, no Content Types are allowed.

https://doc.ibexa.co/en/latest/extending/extending_udw/#configuration

Do we have bug in UDW?

Reply:
@dew326 I think so, when the array is empty all content types are allowed, it behaves exactly the same when `null`.

Reply:
Works for me. 
I believe the problem is that when I set `[]` in the configuration it still gave me the `null` in the configuration.
When I hardcode `[]` in the JS, UDW works as expected, I cannot confirm any content item.

Reply:
The above comments apply to 3.x. In 2.5 we had this by `[null]` - no content types allowed.

Reply:
Changed it to `[null]`.

https://github.com/ezsystems/ezplatform-admin-ui/pull/1666/commits/2e5e5bae271df639a9d53a1cf4bc87978d410251